app/views/targets/_info.json.jbuilder

```diff
@@ -0,0 +1,6 @@
+json.id target.id
+json.topic_id target.topic_id
```

Comment:
I don't know how is made the frontend, but maybe you need return the topic information, no? WDYT? to do that you can use an topic's info partial